packages/host/src/Footer.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {
+  Box,
+  Home,
+  styled,
+  Typography,
+  useHostContext,
+  UserIcon,
+} from '@openmsupply-client/common';
+
+export const Footer: React.FC = () => {
+  const { user, store } = useHostContext();
+  const PaddedCell = styled(Box)({ display: 'flex', padding: '0 8px' });
+  const iconStyles = { color: 'midGrey', height: '16px', width: '16px' };
```

Comment:
ok, so i'm really mixing the style of styles with this. I wanted to use the access to `theme` that `sx` provides and also didn't want to have things duplicated.

Reply:
I think it's fine 🤷   it's unlikely one of these change and not the other?